renderer/src/typings.d.ts

```diff
@@ -62,10 +62,18 @@ export type ActivityEventMessage = {
 export type FILTransactionStatus = 'sent' | 'processing' | 'failed'
 
 export type FILTransaction = {
-  hash: string
-  timestamp: number
-  status: TransactionStatus
-  outgoing: boolean
-  amount: string
-  address: string
+  hash: string;
+  height: number;
+  timestamp: number;
+  status: TransactionStatus;
+  outgoing: boolean;
+  amount: string;
+  address: string;
 }
+
+// helper
+type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>
+
+// FILTransaction with certain properties changed to optional
+type FILTransactionProcessing = PartialBy<FILTransaction, 'hash' | 'height'>
+type FILTransactionLoading = PartialBy<FILTransaction, 'status' | 'timestamp'>
```

Comment:
Same comments as for the typings in `main`.

Is it perhaps time to create a shared typings file that can be used by both `main` and `renderer`?

Reply:
I agree! I've created an issue for this: https://github.com/filecoin-station/filecoin-station/issues/445